R/cnd-signal.R

```diff
@@ -215,6 +215,10 @@ default_message_file <- function() {
   }
 }
 
+is_rstudio <- function() {
+  Sys.getenv("RSTUDIO_SESSION_PID") == Sys.getpid()
```

Comment:
This made me pause until I realised `Sys.getenv()` is uniformly typed (always length 1, always non missing). Maybe use `is_string()` or `identical()` just for explicitness?

Reply:
Yeah, it's a bit weird but I don't see a better way that isn't much more verbose